src/RGBPatternManager.cpp

```diff
@@ -97,12 +97,21 @@ void RGBPatternManager::RunPattern(void* arg) {
 
   while (true) {
     for (const auto& state : pattern) {
+      // WS2812B usually takes commands in GRB order
+      // https://cdn-shop.adafruit.com/datasheets/WS2812B.pdf - Page 5
+      // But some actually expect RGB!
+
+#if defined(OPENSHOCK_LED_FLIP_RG_CHANNELS) && OPENSHOCK_LED_FLIP_RG_CHANNELS
+      // RG Flipped, RGB
+      std::uint8_t g = static_cast<std::uint8_t>(static_cast<std::uint16_t>(state.red) * brightness / 255);
+      std::uint8_t r = static_cast<std::uint8_t>(static_cast<std::uint16_t>(state.green) * brightness / 255);
+#else
+      // Default, GRB
       std::uint8_t r = static_cast<std::uint8_t>(static_cast<std::uint16_t>(state.red) * brightness / 255);
       std::uint8_t g = static_cast<std::uint8_t>(static_cast<std::uint16_t>(state.green) * brightness / 255);
+#endif
```

Comment:
Instead of doing this repetition of logic, could you instead do something like this?

```
#if OPENSHOCK_LED_FLIP_RG_CHANNELS
    std::swap(r,g);
#endif
```